ocfl-java-core/src/main/java/edu/wisc/library/ocfl/core/extension/storage/layout/config/NTupleOmitPrefixStorageLayoutConfig.java

```diff
@@ -0,0 +1,195 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2019 University of Wisconsin Board of Regents
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package edu.wisc.library.ocfl.core.extension.storage.layout.config;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import edu.wisc.library.ocfl.core.extension.storage.layout.NTupleOmitPrefixStorageLayoutExtension;
+import edu.wisc.library.ocfl.api.util.Enforce;
+import edu.wisc.library.ocfl.core.extension.OcflExtensionConfig;
+
+/**
+ * Configuration for the <a href="https://ocfl.github.io/extensions/0007-n-tuple-omit-prefix-storage-layout.md">
+ * N Tuple Storage Layout</a> extension.
+ *
+ * @author vcrema
+ * @since 2021-10-25
+ */
+public class NTupleOmitPrefixStorageLayoutConfig implements OcflExtensionConfig {
+
+	public enum ZeroPadding {
+		  LEFT,
+		  RIGHT
+	}
```

Comment:
I don't think this will serialize to `left` and `right` as is required by the extension.

I _think_ it will work if you do the following, but you should confirm:

```java
public enum ZeroPadding {
    @JsonProperty("left")
    LEFT,
    @JsonProperty("right")
    RIGHT
}
```

Reply:
I could not get the JsonProperty annotation to work properly but I was able to get this working in a different way.